plugin/core/protocol.py

```diff
@@ -156,7 +165,6 @@ class InsertTextMode:
 LocationLink = TypedDict('LocationLink', {
     'originSelectionRange': Optional[RangeLsp],
     'targetUri': DocumentUri,
-    'targetRange': RangeLsp,
```

Comment:
I would not remove `targetRange`, because it exist in the spec.

But I do not see it as a bad thing to remove it for now, if it is not used.

Reply:
I agree that it should not be removed. Somehow I didn't realize that it's a type from the spec.